vochain/transaction/vochaintx/vochaintx.go

```diff
@@ -43,3 +44,12 @@ func (tx *VochainTx) Unmarshal(content []byte, chainID string) error {
 func TxKey(tx tmtypes.Tx) [32]byte {
 	return sha256.Sum256(tx)
 }
+
+// TransferTokensMeta wraps useful information about a token transfer
```

Comment:
Please, make the documentation of the exported types more extense (Copilot can suggest much more) and add dots to the end of the statements.